Formula/mafft.rb

```diff
@@ -0,0 +1,43 @@
+class Mafft < Formula
+  desc "Multiple alignments with fast Fourier transforms"
+  homepage "https://mafft.cbrc.jp/alignment/software/"
+  url "https://mafft.cbrc.jp/alignment/software/mafft-7.313-with-extensions-src.tgz"
+  sha256 "c48e5e05b427cae0d862daaef6148675d5ef57e24425c17b4c3d8da5b060eabd"
+
+  fails_with :clang do
+    build 421
+    cause <<-EOS
```

Comment:
`~EOS` instead of `-EOS`

Reply:
Actually, let's remove the whole `fails_with` block, that clang version is _ancient_